chaosaws/rds/actions.py

```diff
@@ -201,17 +201,3 @@ def describe_db_cluster(cluster_id: str, client: boto3.client) -> AWSResponse:
     except ClientError as e:
         raise FailedActivity('unable to identify cluster %s: %s' % (
             cluster_id, e.response['Error']['Message']))
-
-
-def describe_db_clusters(filters: List[Dict[str, Any]],
```

Comment:
is there a reason why this whole function goes away?

Reply:
Largely because I wrote it in such a way that it's not useable with the existing functions. Currently, RDS is handles by providing the identifier of the resource. There is no function that allows for the adding of `filters`.